packages/react/package.json

```diff
@@ -21,7 +21,7 @@
     "clean": "just-scripts clean",
     "code-style": "just-scripts code-style",
     "codepen": "node ../../scripts/local-codepen.js",
-    "e2e": "yarn workspace @fluentui/react-examples e2e --package react",
+    "e2e": "yarn workspace @fluentui/react-examples e2e",
```

Comment:
these e2e tests can finally be enabled on CI, since the stories don't exist in react-components storybook